experiment/ci-janitor/main.go

```diff
@@ -139,10 +131,11 @@ func main() {
 		logrus.Fatalf("Invalid options: %v", err)
 	}
 
-	conf, err := config.Load(o.configPath, o.jobConfigPath, o.supplementalProwConfigDirs.Strings())
+	agent, err := o.prowConfig.ConfigAgent()
```

Comment:
nit: technically `prowConfig.ConfigAgent` stutters, Go-ism would be `prowConfig.Agent()`